src/app/notifications/toast/toast-style/toast-style.component.scss

```diff
@@ -1,31 +1,43 @@
-@import '~igniteui-angular/lib/core/styles/themes/index';
+.sample-content{
+    margin: 10% auto 0;
+}
+
+.sample-column {
+    min-width: 20vw;
```

Comment:
This actually makes the sample less responsive. Take a look at the stackblitz sample and resize the preview to see how the buttons are not properly positioned.

Reply:
I changed the min-width to 270px (the default value is 300px) so that the three examples can be displayed in one line on smaller screens. Hope it looks better now.